src/lib.rs

```diff
@@ -189,6 +189,26 @@ impl Client {
         }
     }
 
+    /// Publishes a registry credential map to the control interface of the lattice.
+    /// All hosts will be listening and all will overwrite their registry credential
+    /// map with the new information. It is highly recommended you use TLS connections
+    /// with NATS and isolate the control interface credentials when using this
+    /// function in production as the data contains secrets
+    pub async fn push_registries(&self, registries: RegistryCredentialMap) -> Result<()> {
```

Comment:
As this is an overwrite, perhaps `put_registries` would be more descriptive from the crate user's perspective even before reading the doc comment